test/ContentBase.test.js

```diff
@@ -63,76 +63,6 @@ describe('ContentBase', () => {
     });
   });
 
-  describe('test ignoring node_modules folder by Default', () => {
-    beforeAll((done) => {
-      server = testServer.start(config, {
-        contentBase: contentBasePublic,
-        watchContentBase: true,
-      });
-      // making sure that chokidar has read all the files
-      server.contentBaseWatchers[0].on('ready', () => {
-        done();
-      });
-      req = request(server.app);
-    });
-
-    afterAll((done) => {
-      testServer.close(() => {
-        done();
-      });
-    });
-
-    it('Should ignore node_modules & watch bar', (done) => {
-      const watchedPaths = server.contentBaseWatchers[0].getWatched();
-      // check if node_modules folder is not in watched list
-      const folderWatched = !!watchedPaths[
-        path.join(contentBasePublic, 'node_modules')
-      ];
-      expect(folderWatched).toEqual(false);
-      // check if bar folder is in watched list
-      expect(watchedPaths[path.join(contentBasePublic, 'bar')]).toEqual([
-        'index.html',
-      ]);
-
-      done();
-    });
-  });
-
-  describe('test not ignoring node_modules folder', () => {
-    beforeAll((done) => {
-      server = testServer.start(config, {
-        contentBase: contentBasePublic,
-        watchContentBase: true,
-        watchOptions: {
-          ignored: /bar/,
-        },
-      });
-      // making sure that chokidar has read all the files
-      server.contentBaseWatchers[0].on('ready', () => {
-        done();
-      });
-      req = request(server.app);
-    });
-
-    afterAll((done) => {
-      testServer.close(() => {
-        done();
-      });
-    });
-
-    it('Should watch node_modules & ignore bar', (done) => {
-      const watchedPaths = server.contentBaseWatchers[0].getWatched();
-      // check if node_modules folder is in watched list
-      expect(
-        watchedPaths[path.join(contentBasePublic, 'node_modules')]
-      ).toEqual(['index.html']);
-      // check if bar folder is not in watched list
-      const folderWatched = !!watchedPaths[path.join(contentBasePublic, 'bar')];
-      expect(folderWatched).toEqual(false);
-      done();
-    });
-  });
```

Comment:
I this we should rewrite test to check what we watch `node_modules` (to avoid future regressions)